### PR TITLE
Audioread Fallback for .m4a files

### PR DESCRIPTION
**Pull request by TCord**
Adds .m4a to supported filetypes of pb.audioread by exchanging SoundFile with audioread for .m4a files.

### CODE REVIEW

File: paderbox/io/audioread.py [hunk 1]
```diff
@@ -166,22 +167,35 @@ def load_audio(
...
-                    'PCM_16': np.int16,
-                    'FLOAT': np.float32,
-                    'DOUBLE': np.float64,
-                })
-                dtype = mapping[f.subtype]
-
-            frames = f._prepare_read(start=start, stop=stop, frames=frames)
-            data = f.read(frames=frames, dtype=dtype, fill_value=fill_value)
-        signal, sample_rate = data, f.samplerate
+        if path[-3:] == 'm4a':
```
Comment (boeddeker) [inline paderbox/io/audioread.py]: Could you include the `.`? (i.e. `'.m4a'`)

File: paderbox/io/audioread.py [hunk 2]
```diff
@@ -166,22 +167,35 @@ def load_audio(
...
+            with ar.audio_open(
+                    path
+            ) as f:
```
Comment (boeddeker) [inline paderbox/io/audioread.py]: Can you add an assert that start and stop aren't supported? And add a comment, that audioread does not support them.

**Comment by boeddeker** [comment]
Is it possible to download a file from the internet to make a test that we can read m4a files?

File: paderbox/io/audioread.py (hunk 1 above)
Comment (TCord) [reply paderbox/io/audioread.py]: The new version now uses suffix() instead of string slicing

**Comment by TCord** [comment]
Unfortunately I could not find any open source .m4a files to create a test

**Comment by boeddeker** [comment]
Ok, I merged this PR

**Pull request merged**